testsuite/constant_pH.py

```diff
@@ -36,14 +36,18 @@ class ReactionEnsembleTest(ut.TestCase):
     type_A = 1
     type_H = 5
     temperature = 1.0
-    standard_pressure_in_simulation_units = 0.00108
     # avoid extreme regions in the titration curve e.g. via the choice
     # (np.random.random()-0.5)
+#    target_alpha=0.5; # choose target alpha close to 0.5 to get good statistics in a small number of steps
+#    Ka=target_alpha*target_alpha/(1.0-target_alpha)*c0; # determine Ka which corresponds to the desired c0 and alpha
+#    pKa=-np.log10(Ka);
```

Comment:
I did not choose 0.5 because this value was quite robust regarding errors from my experience for typical errors which happened during coding. But of course in principle any point on the curve should be suitable for testing, so I don't have a preference.